### PR TITLE
Droth 40 search by

### PR DESCRIPTION
**Pull request by oappicgi**
Feature to search with link-id, masstransitstop id (national & livi-id) and speedlimit id.

### CODE REVIEW

**Comment by halied** [comment]
OK

File: UI/src/router.js [hunk 1]
```diff
@@ -50,20 +50,30 @@
...
+                eventbus.once('linkProperties:available', function () {
+                  models.selectedLinkProperty.open(response.id);
+                });
+              });
+            }
+            map.setCenter(new OpenLayers.LonLat(response.middlePoint.x, response.middlePoint.y), 12);
+          }
+          else
+          {
+            //might be nice to show error message for user if roadlink  applied to #linkProperty/ url does not exist
```
Comment (marquesfm) [inline UI/src/router.js]: I agree with the idea of the show error message for user if roadlink doesn't exist.
If we don't have the message yet, maybe it's better make this comment like a TODO.

**Comment by joanacoelho** [comment]
I agree with the previous comment, but everything else seems ok.

**Comment by marquesfm** [comment]
With the exception of the comments is all OK for me.

File: UI/src/model/LocationSearch.js [hunk 2]
```diff
@@ -14,30 +22,176 @@
...
+     * @returns {*}
+     */
+    var roadNumberAndSpeedLimitSearch = function(input){
+      var roadNumberSearch = backend.getCoordinatesFromRoadAddress(input);
+      var speedlimitSearch= backend.getSpeedLimitsLinkIDFromSegmentID(input);
+      return $.when(
+        speedlimitSearch, roadNumberSearch).then(function(speedlimitdata,roadData) {
+        var returnObject = roadLocationAPIResultParser(roadData);
+        var linkFound =_.get(speedlimitdata[0], 'success');
+        if (linkFound === true) {
```
Comment (MAC-CGI) [inline UI/src/model/LocationSearch.js]: In this case I thing that you can remove the variable 'link Found' and put directly the content in the if clause, like you have done in line 81 of this file. Do you agree?

**Comment by MAC-CGI** [comment]
In addition of my suggestion, everything seems ok.

File: UI/src/model/LocationSearch.js (hunk 2 above)
Comment (oappicgi) [inline UI/src/model/LocationSearch.js]: Yea, it is probably clear enough that way. Thank you

**Pull request merged**